src/content/contributions/add-ons/add-ons.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Add-ons
+---
+
+import { CheckmarkFilled16 } from '@carbon/icons-react';
+
+<AnchorLinks>
+
+- [Add-on repositories](#add-on-repositories)
+- [Contributing to add-ons](#contributing-to-add-ons)
+- [Ownership and maintenance](#ownership-and-maintenance)
+
+</AnchorLinks>
+
+## Add-on repositories
+
+Add-on repositories, or repos, contain components built for a specific product or experience. The repos are built on top of the core Carbon repo, and enable teams to create their own custom components that follow Carbon's visual style and guidelines.
```

Comment:
> The repos are built on top of the core Carbon repo, and enable teams to create

This sentence seems to imply a hierarchy that may not necessarily be there. Add-on components should be built following the Carbon Design System visual style/guideilnes (namely tokens) but don't necessarily have any connection to the core Carbon repo.

Reply:
👍